src/utils/globalnav.ts

```diff
@@ -14,39 +14,16 @@ export const RIGHT_NAV_LINKS = [
 ];
 
 export const LEFT_NAV_LINKS = [
-  {
-    type: 'DEFAULT',
-    label: 'Learn',
-    url: 'https://amplify.aws/learn',
-    order: 1
-  },
   {
     type: 'DEFAULT',
     label: 'UI Library',
     url: 'https://ui.docs.amplify.aws/',
-    order: 2
+    order: 1
   },
   {
     type: 'DEFAULT',
     label: 'Contribute',
     url: '/contribute/',
-    order: 3
-  }
-];
-
-export const SOCIAL_LINKS = [
```

Comment:
Why are we deleting this? Is it unused?

Reply:
I'm building a staging site and will let everyone know when it's done so we can all check